packages/ocean-react/src/IconButton/__tests__/IconButton.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
I missed these tests with InlineSnapshot as we did in `Button.test.ts`:
- renders element properly
- renders a link icon button